app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -72,6 +92,63 @@ class Identification extends React.Component {
       });
   }
 
+  getLanguageOptions(languageType) {
+    const langOptions = supportedLanguages.languages.map(function(lang) {
+      let fullySupported = lang.supported.sms && lang.supported.email && lang.supported.phone;
+      let langLabel = fullySupported ? lang.name : lang.name + '*';
+      return { value: lang.name, label: langLabel, type: languageType };
+    });
+    return langOptions;
+  }
+
+  getLanguageValue(language, languageType) {
+    return this.getLanguageOptions(languageType).filter(lang => lang.value === language);
+  }
+
+  renderLanguageSupportMessage(selectedLanguage, languageType) {
+    if (selectedLanguage) {
+      let languageJson = supportedLanguages.languages.find(l => l.name === selectedLanguage);
+      let sms = languageJson.supported.sms;
```

Comment:
Same note about `const` vs `let`. I also don't think you need a new variable for sms, email, and phone so if you wanted to join them that'd be fine.